src/models/json-ld.js

```diff
@@ -0,0 +1,352 @@
+import * as R from "ramda";
+
+const ALLOW = "reprolib:terms/allow";
+const ABOUT = "reprolib:terms/landingPage";
+const ALT_LABEL = "http://www.w3.org/2004/02/skos/core#altLabel";
+const AUDIO_OBJECT = "schema:AudioObject";
+const AUTO_ADVANCE = "reprolib:terms/auto_advance";
+const BACK_DISABLED = "reprolib:terms/disable_back";
+const CONTENT_URL = "schema:contentUrl";
+const DELAY = "reprolib:terms/delay";
+const DESCRIPTION = "schema:description";
+const DO_NOT_KNOW = "reprolib:terms/dont_know_answer";
+const ENCODING_FORMAT = "schema:encodingFormat";
+const FULL_SCREEN = "reprolib:terms/full_screen";
+const IMAGE = "schema:image";
+const IMAGE_OBJECT = "schema:ImageObject";
+const INPUT_TYPE = "reprolib:terms/inputType";
+const INPUTS = "reprolib:terms/inputs";
+const IS_ABOUT = "reprolib:terms/isAbout";
+const ITEM_LIST_ELEMENT = "schema:itemListElement";
+const MAX_VALUE = "schema:maxValue";
+const MEDIA = "reprolib:terms/media";
+const MIN_VALUE = "schema:minValue";
+const MULTIPLE_CHOICE = "reprolib:terms/multipleChoice";
+const NAME = "schema:name";
+const PREAMBLE = "reprolib:terms/preamble";
+const PREF_LABEL = "http://www.w3.org/2004/02/skos/core#prefLabel";
+const QUESTION = "schema:question";
+const REFUSE_TO_ANSWER = "reprolib:terms/refused_to_answer";
+const REQUIRED_VALUE = "reprolib:terms/required";
+const SCHEMA_VERSION = "schema:schemaVersion";
+const SCORING_LOGIC = "reprolib:terms/scoringLogic";
+const SHUFFLE = "reprolib:terms/shuffle";
+const TIMER = "reprolib:terms/timer";
+const TRANSCRIPT = "schema:transcript";
+const URL = "schema:url";
+const VALUE = "schema:value";
+const RESPONSE_OPTIONS = "reprolib:terms/responseOptions";
+const VARIABLE_NAME = "reprolib:terms/variableName";
+const JS_EXPRESSION = "reprolib:terms/jsExpression";
+const VERSION = "schema:version";
+const IS_VIS = "reprolib:terms/isVis";
+const ADD_PROPERTIES = "reprolib:terms/addProperties";
+const COMPUTE = "reprolib:terms/compute";
+const MESSAGES = "reprolib:terms/messages";
+const MESSAGE = "reprolib:terms/message";
+export const ORDER = "reprolib:terms/order";
+
+export const languageListToObject = (list) => {
```

Comment:
@ngonzalvez I documented all the components. I haven't documented the JSON-LD and services that I directly copied from the Mobile App, since I am not yet sure of the complete functionality of these files. But As I gradually use these pieces, I will write the documentation.